configuration/routing.tcl

```diff
@@ -13,7 +13,9 @@
 # limitations under the License.
 
 # Routing defaults
-set ::env(ROUTING_CORES) 4
+if {! [info exists ::env(ROUTING_CORES)] } {
+    set ::env(ROUTING_CORES) 2
```

Comment:
Why is the default changed from 4 to 2?  The documentation doesn't reflect that change.

Reply:
No reason. I’ve dealt with my fair share of dual cores.

Reply:
Unless you plan to change the docs I suggest you leave it at 4.  Even 4 seems a bit low for modern computers.

Reply:
Docs have been changed. When running multiple designs in parallel on cloud environments, two cores per design is quite reasonable… people with more resources can simply override the relevant environment variable.